code/modules/events/spacevine.dm

```diff
@@ -499,7 +542,7 @@
 	var/obj/item/seeds/kudzu/seed = new(vine.loc)
 	seed.mutations |= vine.mutations
 	seed.set_potency(mutativeness * 10)
-	seed.set_production(11 - (spread_cap / initial(spread_cap)) * 5) //Reverts spread_cap formula so resulting seed gets original production stat or equivalent back.
+	seed.set_production(11 - (10 * initial(spread_multiplier) / spread_multiplier)) //Reverts spread_multiplier formula so resulting seed gets original production stat or equivalent back.
```

Comment:
if you are going to change balance values here, please use a variable or a define to represent these numbers